### PR TITLE
refactor: Migrate to new extension methods for `String.toInstruction(...)` and `String.toInstructions(...)`

### PR DESCRIPTION
**Pull request by Itroublve**

### CODE REVIEW

**Comment by Itroublve** [comment]
> Is this done? Because it's marked as draft

No, I'm still working on general-ads &  microg-support... I'll mark this as ready for review when I'm done.

File: src/main/kotlin/app/revanced/patches/youtube/layout/minimizedplayback/patch/MinimizedPlaybackPatch.kt [hunk 1]
```diff
@@ -19,19 +18,19 @@ import app.revanced.patches.youtube.layout.minimizedplayback.signatures.Minimize
 @Description("Enable minimized and background playback.")
 @MinimizedPlaybackCompatibility
 @Version("0.0.1")
-class MinimizedPlaybackPatch : BytecodePatch(
+class xMinimizedPlaybackPatch : BytecodePatch(
```
Comment (TheJeterLP) [inline src/main/kotlin/app/revanced/patches/youtube/layout/minimizedplayback/patch/MinimizedPlaybackPatch.kt:21]: Is this a typo here?

Comment (Itroublve) [reply src/main/kotlin/app/revanced/patches/youtube/layout/minimizedplayback/patch/MinimizedPlaybackPatch.kt:21]: Certainly is..

**Comment by oSumAtrIX** [comment]
Ready to get merged?

**Comment by Itroublve** [comment]
Yes

**Pull request merged**